lib/llm/providers/anthropic.rb

```diff
@@ -47,13 +47,14 @@ def complete(prompt, role = :user, **params)
     # @param prompt (see LLM::Provider#transform_prompt)
     # @return (see LLM::Provider#transform_prompt)
     def transform_prompt(prompt)
-      if URI === prompt
+      if LLM::File === prompt
+        file = prompt
         [{
           type: :image,
           source: {
             type: :base64,
```

Comment:
This could be more compact, right ? Do we need to use 7-8 lines to express this ? The `source` Hash could be one line ?